from app.file_exception_handling import filehandling
file_open = filehandling()
print(file_open.exception_handling())